tests/components/application-connector/resources/charts/gateway-test/templates/applications/manual.yml

```diff
@@ -0,0 +1,23 @@
+apiVersion: applicationconnector.kyma-project.io/v1alpha1
+kind: Application
+metadata:
+  name: manual
+  namespace: "{{ .Values.namespace }}"
+spec:
+  description: Endpoints for manual tests
+  skipVerify: true
+  labels:
+    app: manual-tests
+  services:
+    - displayName: oauth-short
```

Comment:
Maybe `oauth-expired-token-renewal`?

Reply:
I was thinking this might be generic for all tests requiring a token with short expire time, but sure, that makes sense too